db/seeds/development.rb

```diff
@@ -139,6 +140,17 @@
             credential_service_providers: [SignIn::Constants::Auth::IDME, SignIn::Constants::Auth::LOGINGOV],
             service_levels: [SignIn::Constants::Auth::LOA3, SignIn::Constants::Auth::IAL2])
 
+# Create VerifiedRepresentative and AccreditedIndividual for logging into accredited_representative_portal
```

Comment:
I would recommend that you don't make this change in the seeds file, just leverage the rake task that you're also creating here